src/AlgebraicGeometry/Schemes/CoveredSchemes/Morphisms/Methods.jl

```diff
@@ -65,3 +65,33 @@ function maps_with_given_codomain(f::AbsCoveredSchemeMorphism, V::AbsSpec)
   return result
 end
 
+########################################################################
+# Comparison
+########################################################################
+function ==(f::AbsCoveredSchemeMorphism, g::AbsCoveredSchemeMorphism)
```

Comment:
If we have `==`, I think we need a hash function?

Reply:
I am not aware of this. Can you elaborate or give a link describing what has to be done here?

Reply:
https://github.com/oscar-system/Oscar.jl/issues/2222

Reply:
We spoke about this a bit today with @fieker . I'm not yet 100% sure how I want to address these issues and would like to leave this for a later cleanup. But thanks for notifying me on this.